sccp/figures/figureS1.py

```diff
@@ -7,50 +7,52 @@
 # GOAL: test Pf2 on lupus data, get visualizations for factor matrices
 
 # load functions/modules ----
-from .common import subplotLabel, getSetup, plotFactors, plotWeight
+from .common import subplotLabel, getSetup, plotFactors, plotWeight, openPf2
 from parafac2 import parafac2_nd
 from ..imports.scRNA import load_lupus_data
-from .common import subplotLabel, getSetup
+from .common import subplotLabel, getSetup, savePf2
 
 
 def makeFigure():
     """Get a list of the axis objects and create a figure."""
     # Get list of axis objects
-    ax, f = getSetup((12, 12), (2, 2))  # fig size  # grid size
+    ax, f = getSetup((12, 16), (2, 2))  # fig size  # grid size
 
     # Add subplot labels
     subplotLabel(ax)
 
-    rank = 5
+    rank = 40
+    group_to_label = "SLE_status" # group to label on left side of factor A plot
 
     (
         lupus_tensor,
         obs,
-    ) = load_lupus_data(every_n = 10)  # so that it builds quickly
+    ) = load_lupus_data()  
 
 
-    weights, factors, _, _ = parafac2_nd(
-        lupus_tensor, rank=rank, n_iter_max=20, random_state=1
-    )
-
     status = (
-        obs[["sample_ID", "SLE_status"]]
+        obs[["sample_ID", group_to_label]]
         .drop_duplicates("sample_ID")
     )
 
     # make sure that these two are in the same order
     bool = status['sample_ID'] == lupus_tensor.condition_labels
     assert bool.mean() == 1.0
 
+    group_labs = status.set_index("sample_ID")[group_to_label]
+
+    #weights, factors, projs, _ = parafac2_nd(
+    #    lupus_tensor, rank=rank, random_state=1, verbose = True
+    #)
 
-    lut = {"SLE": "c", "Healthy": "m"}
-    row_colors = status["SLE_status"].map(lut)
+    #savePf2(weights, factors, projs, dataName = "lupus")
```

Comment:
@pascoesean you can get rid of this line and the import `savePf2`!

Reply:
And the `parafac2_nd` as well!

Reply:
@andrewram4287 I left them in because I was thinking it might be nice to have a record of how we ran it the first time it comes up (which would be here?) But I can (re)move it if you think it makes more sense to be streamlined-- either way doesn't really affect performance

Reply:
We can just get rid of it for now!